cluster-provision/gocli/cmd/run_test.go

```diff
@@ -0,0 +1,64 @@
+package cmd
```

Comment:
There's no particular reason, i'm mostly just familiar with testify. I am very open to using ginkgo if it can solve my need to test for SSH command execution using a mock